chart/values.yaml

```diff
@@ -337,42 +335,6 @@ backfillRetryableErrors:
       cpu: 0
   tolerations: []
 
-cleanDuckdbIndexCache:
-  enabled: true
-  log:
-    level: "info"
-  action: "clean-directory"
-  schedule: "0 */3 * * *"
-  # every 3 hours
-  nodeSelector: {}
-  resources:
-    requests:
-      cpu: 0
-    limits:
-      cpu: 0
-  tolerations: []
-  # the time interval at which cached downloaded data files will be considered as expired and will be deleted
-  expiredTimeIntervalSeconds: 600
-  subfolderPattern: "cache/*"
-
-cleanDuckdbIndexDownloads:
-  enabled: true
-  log:
-    level: "info"
-  action: "clean-directory"
-  schedule: "0 0 * * *"
-  # at 00:00
-  nodeSelector: {}
-  resources:
-    requests:
-      cpu: 0
-    limits:
-      cpu: 0
-  tolerations: []
-  # the time interval at which cached downloaded data files will be considered as expired and will be deleted
-  expiredTimeIntervalSeconds: 600
-  subfolderPattern: "downloads/*"
```

Comment:
currently, are we still using both cache/ and downloads/? Only for the search service?

Reply:
Yes, it is only for search service using the shared storage.